utils/install_dependencies.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+apt install -y cmake ninja-build libboost-program-options-dev libfmt-dev libspdlog-dev
```

Comment:
Shouldn't this file be in /scripts?

Reply:
I don't know, should it?
I thought it is tfw_logger dependencies and it would be better to store the script in the same directory.

Reply:
Yes, it's better to keep shell stuff separately. But only if it isn't the last change and it doesn't take long time